dask-gateway/Dockerfile

```diff
@@ -32,39 +39,40 @@ RUN useradd -m -U -u 1000 dask \
     && echo "aggressive_update_packages: []" >> /home/dask/.condarc \
     && find /opt/conda/ -follow -type f -name '*.a' -delete \
     && /opt/conda/bin/conda clean -afy \
-    && chown -R dask:dask /opt/conda \
-    && apt-get autoremove --purge -y wget bzip2 \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+    && chown -R dask:dask /opt/conda
 
 USER 1000:1000
 ENV PATH="/opt/conda/bin:$PATH"
 WORKDIR /home/dask/
 
 ENTRYPOINT ["tini", "-g", "--"]
 
+
+
 # ** An image with all of dask-gateway's dependencies **
 FROM miniconda as dependencies
 
-ARG DASK_VERSION=2021.7.1
-ARG DISTRIBUTED_VERSION=2021.7.1
+ARG DASK_VERSION=2021.8.1
+ARG DISTRIBUTED_VERSION=2021.8.1
```

Comment:
I just bumped these while I was at it so all the conda/pip packages installed were updated across now that I had to update at least aiohttp.